addons/google_calendar/models/google_sync.py

```diff
@@ -305,11 +305,11 @@ def _create_from_google(self, gevents, vals_list):
     def _get_sync_partner(self, emails):
         normalized_emails = [email_normalize(contact) for contact in emails if email_normalize(contact)]
         user_partners = self.env['mail.thread']._mail_search_on_user(normalized_emails, extra_domain=[('share', '=', False)])
-        partners = [user_partner for user_partner in user_partners if user_partner.type != 'private']
+        partners = list(user_partners)
```

Comment:
is there a point in making a list instead of keeping a recordset? As we add more records afterwards, maybe we could use sets instead?